whelk-core/src/main/groovy/whelk/search/ESQuery.groovy

```diff
@@ -23,10 +23,11 @@ class ESQuery {
     private static final ObjectMapper mapper = new ObjectMapper()
     private static final int DEFAULT_PAGE_SIZE = 50
     private static final List RESERVED_PARAMS = [
-        'q', 'o', '_limit', '_offset', '_sort', '_statsrepr', '_site_base_uri', '_debug', '_boost', '_lens'
+        'q', 'o', '_limit', '_offset', '_sort', '_statsrepr', '_site_base_uri', '_debug', '_boost', '_lens', '_suggest'
     ]
     private static final String OR_PREFIX = 'or-'
     private static final String EXISTS_PREFIX = 'exists-'
+    private static final List SUGGEST_LANGS = ['sv', 'en']
```

Comment:
Use existing `ElasticSearch.LANGUAGES_TO_INDEX` instead?

Reply:
Ah, good catch, forgot about that!